config.go

```diff
@@ -608,6 +608,14 @@ func (s *Settings) CreateInstallationToken(ctx context.Context, owner, repo stri
 	return token.GetToken(), nil
 }
 
+func (s *Settings) CreateInstallationTokenWithDefaultOwnerRepo(ctx context.Context, opts *github.InstallationTokenOptions) (string, error) {
+	if s.AuthConfig.Github == nil || s.AuthConfig.Github.DefaultOwner == "" || s.AuthConfig.Github.DefaultRepo == "" {
+		// TODO EVG-19966: Return error here
```

Comment:
Can you explain this todo? You are returning an error

Reply:
sorry that was there just so I could debug something but we shouldn't have it at the moment